packaging/app.organicmaps.desktop.metainfo.xml

```diff
@@ -115,6 +115,27 @@
   </screenshots>
 
   <releases>
+   <release version="2024.08.15-21" date="2024-08-15">
+     <description>
+       <p>Highlights:</p>
+         <ul>
+           <li>OpenStreetMap data as of August 10</li>
+           <li>Improved routing in cases of bad surface and missing speed limit</li>
+           <li>Display road shields in India</li>
+           <li>Self-service and outdoor seating support</li>
+           <li>Use NotoSans font instead of Serif for Hebrew</li>
```

Comment:
• Traffic light icons are displayed in car navigation mode
• Selected places with WiFi show emoji symbol 🛜
• Fixed wrong selection of a bookmarked place when tapping on its title
• Disabled map rotation snapping to the north angle
• Properly export modified bookmark names in GPX
• Fixed import of KMB files

Reply:
Done